learners/setup.md

```diff
@@ -83,29 +83,32 @@ If you run into any difficulties, please request help before the workshop begins
 
 [Conda](https://docs.conda.io/projects/conda/en/latest/) is the package management system associated with [Anaconda](https://anaconda.org) and runs on Windows, macOS and Linux.
 Conda should already be available in your system once you installed Anaconda successfully. Conda thus works regardless of the operating system.
-Make sure you have an up-to-date version of Conda running.
+
+1. Make sure you have an up-to-date version of Conda running.
 See [these instructions](https://docs.anaconda.com/anaconda/install/update-version/) for updating Conda if required.
 {: .callout}
 
-To create a conda environment called `dl_workshop` with the required packages, open a terminal (Mac/Linux) or Anaconda prompt (Windows) and type the command:
+2. To create a conda environment called `dl_workshop` with the required packages, open a terminal (Mac/Linux) or Anaconda prompt (Windows) and type the command:
 ```bash
 conda create --name dl_workshop python jupyter seaborn scikit-learn pandas
```

Comment:
Instead of upgrading seaborn in an additional step, I would add the requirement of seaborn>= here. If this becomes more complicated we could also think of supplying an `environment.yml` file.
```suggestion
conda create --name dl_workshop python jupyter 'seaborn>=13.0.0' scikit-learn pandas
```

Reply:
Makes sense! Thanks @svenvanderburg.